org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/handlers/BundleUtils.java

```diff
@@ -267,6 +268,10 @@ private static IStatus startBundles(Collection<Bundle> bundles) {
 				status.add(new Status(IStatus.ERROR, context.getBundle().getSymbolicName(), "Could not start: " + bundle.getSymbolicName() + '(' + bundle.getLocation() + ':' + bundle.getBundleId() + ')' + ". It's state is uninstalled."));
 				continue;
 			}
+			if ((bundle.adapt(BundleRevision.class).getTypes() & BundleRevision.TYPE_FRAGMENT) != 0) {
+				continue;

```

Comment:
Could we add a warning for this ? Something like "Fragment bundle $BSN ( $LOCATION ) cannot be started. Skipping."

Reply:
@rgrunber A warning without resolution is not helpful for users. Can this be a debug log statement? Is there a recommended pattern?

Reply:
Yes, I think this is fine. It would show up in the Eclipse log file.